include/fastdds-monitor/backend/SyncBackendConnection.h

```diff
@@ -292,15 +292,20 @@ class SyncBackendConnection
             DataKind data_kind,
             EntityId source_entity_id,
             EntityId target_entity_id,
-            uint16_t bins,
-            Timestamp start_time,
-            Timestamp end_time,
-            StatisticKind statistic_kind);
+            uint16_t bins = 0,
+            StatisticKind statistic_kind = StatisticKind::NONE,
```

Comment:
Why have you change the `get_data()` signature? I'd keep the `SyncBackendConnection::get_data()` signature the same as the `StatisticsBackend::get_data()`